package.json

```diff
@@ -90,6 +90,7 @@
     "pre-git": "3.17.1",
     "rimraf": "3.0.0",
     "shx": "0.3.3",
+    "stream-json": "1.7.4",
```

Comment:
this should go in `modules/cudf/package.json`

Reply:
Removed.